src/dispatch-payload.ts

```diff
@@ -142,3 +125,23 @@ function getFunctionCallbackID(
         ?.function_data?.function?.callback_id ?? "";
   }
 }
+
+export function extractBaseHandlerArgsFromPayload(
+  payload: InvocationPayload<ValidInvocationPayloadBody>,
+): BaseHandlerArgs {
+  const { body, context } = payload;
+  const env = context.variables || {};
+  const team_id = context.team_id || "";
+  const enterprise_id = body.enterprise_id ||
+    (body as BaseEventInvocationBody).enterprise?.id || "";
```

Comment:
The extraction of `enterprise_id` prior to this PR was done differently for different events, which I tried to capture in this line. The interactivity events looked for `body.enterprise.id` whereas the `function_executed` event looked for `body.enterprise_id`. I see no reason why we can't look for any of these fields here?